pyproject.toml

```diff
@@ -29,7 +29,10 @@ enabled = true
 # version = "0.0.1"
 
 [project.urls]
+Documentation = "https://github.com/dafnifacility/cli/blob/main/docs/dafni_cli.md"
+"Release notes" = "https://github.com/dafnifacility/cli/releases"
 Source = "https://github.com/dafnifacility/cli"
+"Issue Tracker" = "https://github.com/dafnifacility/cli/issues"
```

Comment:
Everything could have quotes, in this case its just because there is a space involved so it isn't valid otherwise, but for consistency I will just add them to the rest of the URLs section.

Reply:
Oh, of course, that makes sense. But the consistency makes me happy - thanks!